scripts/eslint/baseConfig.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+const eslintrc = require('../../.eslintrc');
+
+const OFF = 0;
+const ERROR = 2;
+
+module.exports = {
+  config: eslintrc,
+  OFF,
```

Comment:
I think this is a bit confusing. Let's just copy and paste these constants, and completely remove this file :-) ESLint also actually accepts strings so we can later remove them altogether.

Reply:
ok